core/src/main/java/io/questdb/griffin/engine/orderby/LimitedSizeSortedLightRecordCursorFactory.java

```diff
@@ -171,6 +181,19 @@ public boolean recordCursorSupportsRandomAccess() {
         return true;
     }
 
+    @Override
+    public void toPlan(PlanSink sink) {
+        sink.type("Sort light");
+        if (loFunction != null) {
+            sink.meta("lo").val(loFunction);
```

Comment:
nit: `loFunction` is not nullable, it defaults to `LongConstant.ZERO` so when `limit` is no specified, `lo` is output in plan

Reply:
I'll remove the 'if (loFunction != null)' condition but can you give me an example where limit lo: 0 is shown even though user doesn't specify limit . 
This factory should only be used when there's order by + explicit limit clause (and limit requires lo at least).

Reply:
![image](https://user-images.githubusercontent.com/7276403/211644038-7313e795-1a15-4460-a376-6a310d1a251e.png)

Reply:
canBeOptimized()  method doesn't allow passing hi=0,lo=null to LimitedSizeSortedLightRecordCursorFactory .  
When limit is not specified then plan will only show "Sort light" or "sort" .
```sql
create table a ( s symbol index) ;
explain select * from a order by s;
--returns 
--Sort light
--  keys: [s]
--    DataFrame
--        Row forward scan
--        Frame forward scan on: a

explain select * from a limit 0
--returns
--Limit lo: 0
--    DataFrame
--        Row forward scan
--        Frame forward scan on: a

explain select * from a order by s desc limit 0
--Sort light lo: 0
--  keys: [s desc]
--    DataFrame
--        Row forward scan
--        Frame forward scan on: a
```

So both standalone limit and sort + limit can show lo: 0 when requested by user (even though it makes no sense), but should skip processing and return quickly.

Reply:
I think this is ok on balance. I can't see us raising eyebrows here when debugging SQL. What do you think?

Reply:
I reckon it could be a tiny bit better to recognize lo=0,hi not set case and return EmptyTableRecordCursorFactory but it's most likely a user error and very unlikely to happen .